src/site/content/en/handbook/content-types/index.md

```diff
@@ -33,14 +33,14 @@ description: |
 ## Codelabs
 **Format:** Multi-step instructions paired with an embedded [sample app](/handbook/markup-sample-app)
 
-**Typical use case:** showing how to do small, fairly discrete development tasks
```

Comment:
discrete is often confused with discreet, so I'd avoid.

Reply:
Fixed in Google Doc